mpyg321/mpyg321.py

```diff
@@ -38,6 +38,47 @@ class MPyg321PlayerError(RuntimeError):
     pass
 
 
+class MPyg321PlayerFileError(MPyg321PlayerError):
+    """Errors encountered by the player related to files"""
+    pass
+
+
+class MPyg321PlayerCommandError(MPyg321PlayerError):
+    """Errors encountered by the player related to player commands"""
+    pass
+
+
+class MPyg321PlayerArgumentError(MPyg321PlayerError):
+    """Errors encountered by the player related to arguments for commands"""
+    pass
+
+
+class MPyg321PlayerEQError(MPyg321PlayerError):
+    """Errors encountered by the player related to the equalizer"""
+    pass
+
+
+class MPyg321PlayerSeekError(MPyg321PlayerError):
+    """Errors encountered by the player related to the seek"""
+    pass
+
+
+mpg_errors = {
```

Comment:
For clarity of code, I think it makes more sense to have an array of objects like that:
```
[
  {
    message: "Error opening stream:",
    error: MPyg321PlayerFileError
  }, 
  {...}
]
```

Reply:
I would also put this variable at the top of the file with the constants, in order to make the code more readable (consts first, then classes)

Reply:
Done, I also moved the error classes to the top of the file (as they need to be defined before they are used). I didn't use the term `error`, instead I chose `exception`, I hope that's okay.